### PR TITLE
add check-freeradius-radclient.rb; bump version

### PR DESCRIPTION
**Pull request by gsfjohnson**
## Pull Request Checklist

**Is this in reference to an existing issue?**

No.
#### General
- [√] Update Changelog following the conventions laid out on [Keep A Changelog](http://keepachangelog.com/)
- [√] Update README with any necessary configuration snippets
- [?] Binstubs are created if needed
- [√] RuboCop passes
- [?] Existing tests pass 
#### New Plugins
- [?] Tests
- [√] Add the plugin to the README
- [√] Does it have a complete header as outlined [here](http://sensu-plugins.io/docs/developer_guidelines.html#coding-style)
#### Purpose

Check entire stack by sending AAA request to server using radclient.
#### Known Compatablity Issues

None.

### CODE REVIEW

File: bin/check-freeradius-radclient.rb [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+  option :help,
+         short: '-h',
+         long: '--help',
+         description: 'Check MySQL replication status',
+         on: :tail,
+         boolean: true,
+         show_options: true,
+         exit: 0
+
```
Comment (eheydrick) [inline bin/check-freeradius-radclient.rb:74]: help option isn't needed as it comes with the CLI library.

File: bin/check-freeradius-radclient.rb [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+  option :secret,
+         short: '-s',
+         long: '--secret VALUE',
+         description: 'RADIUS secret'
+
+  option :timeout,
+         short: '-t',
+         long: '--timeout VALUE',
+         description: 'RADIUS timeout',
+         default: 1
```
Comment (eheydrick) [inline bin/check-freeradius-radclient.rb]: add a to_i proc here to ensure the value is an int.

File: bin/check-freeradius-radclient.rb [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+  option :retry,
+         short: '-r',
+         long: '--retry VALUE',
+         description: 'RADIUS retries',
+         default: 1
```
Comment (eheydrick) [inline bin/check-freeradius-radclient.rb]: ditto on the proc here

**Comment by eheydrick** [comment]
This is cool. I left a few comments, can you take a look.

**Comment by gsfjohnson** [comment]
Thanks for the feedback- all good suggestions.  Changes committed.

**Comment by eheydrick** [comment]
Thanks!

**Comment by eheydrick** [comment]
Released in 0.1.0.

**Pull request merged**